client/jetpack-onboarding/connect-success.jsx

```diff
@@ -37,6 +37,7 @@ class ConnectSuccess extends PureComponent {
 						image={ illustration }
 						onClick={ onClick }
 						href={ href }
+						e2eType={ 'continue' }
```

Comment:
Nitpick, with JSX, it's actually preferred to use double quotes for string literals:

```js
e2eType="continue"
```

Also, we try to list props in alphabetic order 😁 

(No need to change, I'm currently rebasing #22849 and will do it there.)

Reply:
Thanks!